orbit/pkg/table/find_cmd/find_cmd_darwin.go

```diff
@@ -0,0 +1,146 @@
+//go:build darwin
+// +build darwin
+
+// Package find_cmd implements a table that executes the /usr/bin/find command.
+// This table provides only a subset of the find functionality. Currently only
+// allows setting the -perm and -type arguments.
+//
+// NOTE(lucas): Why does this table exist?
+// Initially we implemented queries that used the osquery core `file` table,
+// but when processing a high number (10k+) of files it exceeded osquery
+// default CPU and memory limits.
```

Comment:
nit: consider simpler msg.
_"This table is using /usr/bin/find and is doing the same thing that osquery core `file` table can do, but with significantly better performance when dealing with high number of files (10k+)"_

Reply:
> "This table is using /usr/bin/find and is doing the same thing that osquery core file table can do, but with significantly better performance when dealing with high number of files (10k+)"

It doesn't do the same thing. The `file` table is more powerful.
Let me know if we are against using `NOTE(user)` and I'm ok with removing that.

Reply:
No problem with the note, just with describing the history 🙂  (Initially .. then ...)
suggestion:
"This table implements the existing osquery **find** functionality (in 'file' table) with significantly better performance when dealing with large number of files (10k+)"

Reply:
OK. What do you mean by "existing osquery find functionality"?

Reply:
in the `file` table

Reply:
> This table implements the existing osquery file functionality 

Sorry, but it doesn't. This table is just a wrapper around the `find` command with a subset of the functionality, that is useful for CIS queries (around permissions for directories).

Reply:
NVM. was for consideration, not strongly against it.